atmos_model.F90

```diff
@@ -1531,6 +1550,10 @@ subroutine assign_importdata(rc)
     real(kind=GFS_kind_phys)                           :: tem, ofrac
     logical found, isFieldCreated, lcpl_fice
     real (kind=GFS_kind_phys), parameter :: z0ice=1.1    !  (in cm)
+!
+      real(kind=GFS_kind_phys), parameter :: himax = 8.0      !< maximum ice thickness allowed
+!     real(kind=GFS_kind_phys), parameter :: himin = 0.1      !< minimum ice thickness required
+      real(kind=GFS_kind_phys), parameter :: hsmax = 2.0      !< maximum snow depth allowed
```

Comment:
Moorthi,, I am curious, why do you add the maximal values for the hice and snow depth? Are the uncapped fields causing some issues?

Reply:
I checked the maximum ice and snow thickness in the first 8 P6 dates (2012,2013 jan,apr,jul and oct) and find no evidence that snow or ice thickness is unreasonable. Snow thickness is generally < 1.0m and ice thickness < 6.5. CICE does have some ridging parameterization so I'm not sure I'd consider that unreasonable. This is only 8 cases of course but I am curious why you might want to impose this.